apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/chronoLocalDateTimeAssertions.kt

```diff
@@ -0,0 +1,17 @@
+package ch.tutteli.atrium.api.fluent.en_GB.jdk8
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.builders.chronoLocalDateTime
+import java.time.chrono.ChronoLocalDateTime
+
+/**
+ * Expects that the subject of the assertion is before [expected] [ChronoLocalDateTime].
+ *
+ * @return This assertion container to support a fluent API.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.9.0
+ */
+fun <T : ChronoLocalDateTime> Expect<T>.isAfter(expected: T): Expect<T> =
```

Comment:
Note, as written in the issue you have to use a type parameter for ChronoLocalDateTime. Hence it should look as follows:
```suggestion
fun <D: ChronoLocalDate, T : ChronoLocalDateTime<D>> Expect<T>.isAfter(expected: T): Expect<T> =
```
You will have to add this additional type parameter also in other places.